api/utils.py

```diff
@@ -96,3 +98,37 @@ def write(self, value):
 
 def bad_request(message):
     return JsonResponse({"statusCode": 400, "error_message": message}, status=400)
+
+
+def generate_field_report_title(
+    country: models.Model,
+    dtype: models.Model,
+    event: models.Model,
```

Comment:
We should use proper model instead of base class models.Model

If there is import error, we can use
```
if typing.TYPE_CHECKING:
    from api.models import Country


....

    country: 'Country',
```